src/handlers/register.rs

```diff
@@ -67,7 +60,13 @@ pub async fn register_submission_handler(
         return (StatusCode::OK, Html(form).into_response());
     }
 
-    // TODO: insert user into database
-
-    (StatusCode::OK, Html("Hi").into_response())
+    if (user_repository.create(&payload.into()).await).is_ok() {
+        let mut response = Html(LoginPage {}.render().unwrap()).into_response();
+        response
+            .headers_mut()
+            .insert("HX-Redirect", HeaderValue::from_static("/login"));
+        (StatusCode::OK, response)
+    } else {
+        (StatusCode::OK, Html("HI").into_response())
```

Comment:
Ještě je otázka, jak cheme řešit tento případ? Napadá mě mít speciální template na 500 server error a ten uživateli vyrenderovat. @vaclavj

Reply:
Jo, to by asi davalo zmysel. Este je otazka ze ci dava zmysel sa snazit nejak vyrenderovat nejaky popis k danemu Erroru alebo proste vyhodit nejaku genericku chybovu hlasku

Reply:
Usera by nemělo zajímat, co na serveru se pokazilo, udělal jsem to genericky